release.config.js

```diff
@@ -0,0 +1,21 @@
+module.exports = {
+  debug: true,
```

Comment:
what does the debug flag do? Could you please add a comment?

Reply:
done